internal/osbuild2/osap_remediation_stage.go

```diff
@@ -0,0 +1,46 @@
+package osbuild2
+
+type OscapRemediationStageOptions struct {
+	DataDir string      `json:"data_dir,omitempty"`
+	Config  OscapConfig `json:"config"`
+}
+
+func (OscapRemediationStageOptions) isStageOptions() {}
+
+func NewOscapRemediationStage(options *OscapRemediationStageOptions) *Stage {
+	return &Stage{
+		Type:    "org.osbuild.oscap.remediation",
+		Options: options,
+	}
+}
+
+func NewOscapRemediationStageOptions(dataDir *string, options OscapConfig) *OscapRemediationStageOptions {
+	return &OscapRemediationStageOptions{
+		DataDir: *dataDir,
+		Config: OscapConfig{
+			ProfileID:    options.ProfileID,
+			Datastream:   options.Datastream,
+			DatastreamID: options.DatastreamID,
+			XCCDFID:      options.XCCDFID,
+			BenchmarkID:  options.BenchmarkID,
+			ArfResult:    options.ArfResult,
+			HtmlReport:   options.HtmlReport,
+			VerboseLog:   options.VerboseLog,
+			VerboseLevel: options.VerboseLevel,
+		},
+	}
+}
+
+type OscapConfig struct {
+	Datastream   string `json:"datastream,omitempty" toml:"datastream,omitempty"`
+	ProfileID    string `json:"profile_id,omitempty" toml:"profile_id,omitempty"`
```

Comment:
These are required by the stage in osbuild so they shouldn't be marked `omitempty`.